src/index.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400&display=swap');
```

Comment:
Can we avoid using the Google Fonts API and its unnecessary tracking?
I suggest uploading the font to a static GCloud bucket.

Reply:
It won't matter if we do it just for this project, we have the same call in the wordpress, the map, carrier-portal... and we have the tag manager in all as well :/